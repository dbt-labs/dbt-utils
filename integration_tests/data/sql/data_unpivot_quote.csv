Customer_Id,Created_At,sTaTuS,SEGMENT,Name
123,2017-01-01,active,tier 1,name 1
234,2017-02-01,active,tier 3,name 3
567,2017-03-01,churned,tier 2,name 2
