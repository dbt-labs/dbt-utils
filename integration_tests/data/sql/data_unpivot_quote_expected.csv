Customer_Id,Created_At,Prop,Val
123,"2017-01-01","SEGMENT","tier 1"
123,"2017-01-01","sTaTuS","active"
234,"2017-02-01","SEGMENT","tier 3"
234,"2017-02-01","sTaTuS","active"
567,"2017-03-01","sTaTuS","churned"
567,"2017-03-01","SEGMENT","tier 2"
